package/com.unity.formats.usd/Tests/Runtime/AnimatedUSDFixture.cs

```diff
@@ -0,0 +1,173 @@
+using System;
```

Comment:
This file is missing the copyright notice

Reply:
What's Unity policy regarding copyrights? Is there anything specific about this repo because of the Apache license?